src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -217,8 +212,8 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
 
             // ReturnsVoid property is overridden in this class so
             // returnsVoid argument to MakeFlags is ignored.
-            this.MakeFlags(methodKind, declarationModifiers, returnsVoid: false, isExtensionMethod: false, isNullableAnalysisEnabled: isNullableAnalysisEnabled,
-                isMetadataVirtualIgnoringModifiers: isExplicitInterfaceImplementation && (declarationModifiers & DeclarationModifiers.Static) == 0);
+            this.MakeFlags(methodKind, _property.RefKind, declarationModifiers, returnsVoid: false, hasAnyBody: hasBody, isExpressionBodied: isExpressionBodied, isExtensionMethod: false, isNullableAnalysisEnabled: isNullableAnalysisEnabled,
```

Comment:
>hasBody [](http://example.com/codeflow?start=112&length=7)

It doesn't look like this parameter represents that we have any body. I think we should use ```(hasBody || hasExpressionBody)``` instead.

Reply:
indeed.  this was why i recommended cleaning this up and using names consistently, and did that work in the original set of changes here.  Inconsistency is a great way for bugs to creep in.  I hope we'll be able to take those changes later, as they will help avoid these subtle bugs and help prevent them from creeping in in the future.

Reply:
Fixed.